### PR TITLE
Add ceph-base bundle

### PR DESCRIPTION
**Pull request by ChrisMacNaughton**

### CODE REVIEW

File: development/ceph-base/bundle.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+relations:
+- - ceph-osd:mon
+  - ceph-mon:osd
+series: xenial
+services:
+  ceph-mon:
+    annotations:
+      gui-x: '750'
+      gui-y: '500'
+    charm: cs:~openstack-charmers-next/xenial/ceph-mon
```
Comment (javacruft) [inline development/ceph-base/bundle.yaml]: /xenial is surplus with multi-series charms.

File: development/ceph-base/bundle.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      expected-osd-count: 3
+    to:
+    - 'lxc:0'
+    - 'lxc:1'
+    - 'lxc:2'
+  ceph-osd:
+    annotations:
+      gui-x: '1000'
+      gui-y: '500'
+    charm: cs:~openstack-charmers-next/xenial/ceph-osd
```
Comment (javacruft) [inline development/ceph-base/bundle.yaml]: /xenial is surplus with multi-series charms.

**Comment by ryan-beisner** [comment]
Can you please also propose a basic README.md, similar to the openstack-base readme.  We need to describe basic requirements of using the bundle, and give a brief usage example.

Ex: https://github.com/openstack-charmers/openstack-bundles/blob/master/development/openstack-base-xenial-newton/README.md

File: development/ceph-base/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Basic OpenStack Cloud
```
Comment (ryan-beisner) [inline development/ceph-base/README.md]: Not an OpenStack Cloud, please update.

File: development/ceph-base/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+ - A minimum of 8GB of physical RAM.
+ - Enough CPU cores to support your capacity requirements.
+ - Two disks (identified by /dev/sda and /dev/sdb); the first is used by MAAS for the OS install, the second for Ceph storage.
+
+## Components
+ - 3 Nodes for Ceph OSDs
+ - 3 Ceph monitors in LXD containers on the OSD machines
+
+All physical servers (not LXC containers) will also have NTP installed and configured to keep time in sync.
```
Comment (ryan-beisner) [inline development/ceph-base/README.md]: Please use consistent terminology (LXD).

**Pull request merged**